test/integration/ggrc/bulk_operations/test_matrix.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (C) 2020 Google Inc.
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+
+"""Tests for module provides endpoints to calc cavs in bulk"""
+
+import json
+import ddt
```

Comment:
Please separate standard and third-party imports by an empty line as per style gude.
```suggestion
import json

import ddt
```

Reply:
Done, thx